### ISSUE TITLE
limel-select: focused option (first option, by default) isn't highlighted when hovered

### ISSUE THREAD

**Issue by adrianschmidt** [Lundalogik/lime-elements#1535]
## Current behavior

The first option doesn't get a hover highlight. The item has a focus highlight when the dropdown is first opened, but that fades out, and then there's no highlight at all when hovering the item. All the other options are highlighted as expected when hovered.

https://user-images.githubusercontent.com/154725/151982291-fda5062a-95d7-42d6-9c57-c72acc5a17dd.mov

1. Click on a limel-select, so that it opens.
2. Hover the available options.

- The currently focused option (the first one by default), isn't highlighted when hovered.

## Expected behavior

The focused option should be highlighted when hovered.

## Environment

- **lime-elements** version: 35.0.0-next.1
- Framework used: Reproducible in the docs.
- Logs:

**Comment by LucyChyzhova** [comment]
I have noticed that some parts of the code have an influence on the possibility to `hover` the first option on the options list. It feels like `tabindex` is involved here. Please take a look at the video. The same list component works fine in the `picker` and of course, the `limel-list` doesn't have this buggy behavior. 

https://user-images.githubusercontent.com/50618208/204218612-3abf026e-37fc-453f-b208-6989caeb124d.mov

I would love to get some dev help here. Until it, I'm putting it back in todo column.

**Comment by Kiarokh** [comment]
I made a PR, which gets rid of the ripple effect from MDC. It fixes the **visual** bugs that are mostly noticeable in this issue, and closes a few other issues regarding similar visual bugs in lists.

However, we still have the `tabindex` problem described here https://github.com/Lundalogik/lime-elements/issues/1468 , which still makes keyboard navigation very messy.

While fiddling around with these visual bugs, I made another issue: https://github.com/Lundalogik/lime-elements/issues/1979 which I think makes the code way less, and life a bit easier.
